src/Elasticsearch.Extensions.Logging/ShipToOptions.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using Elastic.Ingest;
+
+namespace Elasticsearch.Extensions.Logging
+{
+	public class ShipToOptions
+	{
+		/// <summary>
+		/// Gets or sets the API Key, where connection pool type is Cloud, and authenticating via API Key.
+		/// </summary>
+		public string ApiKey { get; set; } = "";
+
+		/// <summary>
+		/// Gets or sets the cloud ID, where connection pool type is Cloud.
+		/// </summary>
+		public string CloudId { get; set; } = "";
+
+   		/// <summary>
```

Comment:
spaces

Reply:
I'm usually using Rider and VS Code these days, mostly on Ubuntu, although sometimes on Windows. (Both should support editor config though).

I cleaned the files in Rider, and then also did a clean on all the other files I touched in the branch/PR.  This did reorganise some of the files, as my code cleanup is configured to put members in alphabetical order (with public before private).

The specific configuration I use for Rider/Resharper is here: https://github.com/sgryphon/csharp-layout  (I usually use the StyleCop layout, allowing the standard rules to be turned on for checking during build, etc).